BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/Implementation/RichPayloadEventSourceTest.cs

```diff
@@ -297,8 +316,8 @@ public void RichPayloadEventSourceOperationStartStopTest()
                     // Expect exactly two events (start and stop)
                     var actualEvents = listener.Messages.Where(m => m.Keywords.HasFlag(RichPayloadEventSource.Keywords.Operations)).Take(2).ToArray();
 
-                    VerifyOperationEvent(requestTelemetry, RequestTelemetry.TelemetryName, EventOpcode.Start, actualEvents[0]);
-                    VerifyOperationEvent(requestTelemetry, RequestTelemetry.TelemetryName, EventOpcode.Stop, actualEvents[1]);
+                    VerifyOperationEvent(requestTelemetry, ServiceProfilerNameContracts[requestTelemetry.GetType()], EventOpcode.Start, actualEvents[0]);
```

Comment:
@xiaomi7732, @SergeyKanzhelev , this is new behavior here and on lines 357-360 below. Tests passed in PR but failed locally on these checks. It looks like tests expected that when Request and Dependency are used in StartOperation API they always yield "operation" as event name instead of "Request" and "RemoteDependency".

However, somewhere along our changes (unknown ATM, tests on PRs were passing regardless...) we changed this behavior and now if outside operation is RequestTelemetry its event name is still "Request" in ETW and not "Operation". Please let me know how important it is and what it the expected behavior here.